master/networking/dual-stack.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Enable dual stack
+canonical_url: 'https://docs.projectcalico.org/master/networking/dual-stack'
+---
+
+### Big picture
+
+Arrange for each Kubernetes pod to get both IPv4 and IPv6 addresses, so that it can communicate over
+both IPv4 and IPv6.
+
+### Value
+
+Communication over IPv6 is increasingly desirable, and the natural approach for cluster pods is to
```

Comment:
Communication over IPv6 is increasingly desirable, and the natural approach for cluster pods is to
be IPv6-native themselves, while still supporting IPv4.